modular_citadel/code/modules/reagents/reagent container/hypovial.dm

```diff
@@ -107,26 +120,26 @@
 	name = "bluespace large hypovial"
 	volume = 120
 	icon_state = "hypoviallargebs"
-	obj_flags = 0
+	unique_reskin = null
 
 
 /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/bicaridine
-	name = "hypovial (bicaridine)"
+	name = "red hypovial (bicaridine)"
```

Comment:
Why do we need to rename every vial with its color? Users are able to use a pen on them to rename as they want and the preloaded vials don't need to be given this extra.

Reply:
It's a small worldbuilding detail, in this case. The vials follow the naming scheme of the following production sequence: build the vial, recolour the vial, use a labeler on it with the chem.